package.json

```
@@ -20,6 +20,7 @@
   "private": true,
   "scripts": {
     "build": "yarn workspace @openfun/verna build",
+    "build:playground": "yarn workspace verna-examples-playground build",
```

Like for `dev:example`, I'll make this command more generic
```suggestion
    "build:example": "yarn workspace ${VERNA_EXAMPLE:-verna-examples-playground} build",
```